.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+**Description:** Describe in a couple of sentence what this PR adds
+
+**JIRA:** Link to JIRA ticket
+
+**Reviewers:**
+- [ ] tag reviewer 
+- [ ] tag reviewer 
```

Comment:
Is it intentional for there to always be 2 reviewers?

Reply:
Please no :)

Reply:
It was just an example from the cookiecutter.  I'll try to make it more clear.